docs/product-user-guide/src/main/asciidoc/chap-process-designer.adoc

```diff
@@ -67,18 +59,18 @@ Due to BPMN2 type restrictions, it is _not_ recommended to use multibyte encodin
 +
 .Opening Variable Editor
 image::3415.jpg[Variable definition input field with the arrow for calling the editor.]
-. Define the process properties on the tab by clicking individual entries. For entries that require other input that just string input, the respective editors can be used by clicking the arrow icon. Note that editors for complex fields mostly provide validation and auto-completion features.
+. Define the process properties on the tab by clicking individual entries. For entries that require other input than just string input, the respective editors can be used by clicking the arrow icon. Note that editors for complex fields mostly provide validation and auto-completion features.
```

Comment:
"Define the process properties on the tab by clicking individual entries.": Can we name the tab?